pyproject.toml

```diff
@@ -20,15 +20,15 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dependencies = [
-  "pydantic~=1.10.8",
-  "jsonschema==3.2.0",
+  "pydantic~=1.10",
+  "jsonschema~=3.0",
   "PyYAML~=6.0",
-  "more-itertools==8.10.0",
-  "Jinja2==3.1.2",
-  "click>=7.1.2",
-  "python-dateutil==2.8.2",
-  "importlib_metadata==6.6.0",
-  "typing-extensions~=4.6.1",
+  "more-itertools~=8.0",
+  "Jinja2~=3.0",
+  "click>=7.0",
```

Comment:
We should probably cap `click` if we're specifying `>=` I _think_ we support `8.x` so perhaps `"click>=7.0,<9.0"`

Reply:
I don't have much context on the `click` dependency, but made that update.